refactor: synch dynamic fees only when `InputFee` is active

j-a-m-l opened the pull request:
## Proposed changes
These changes avoid making requests at the beginning to fetch the fee statistics of every network, and then, periodically to have them updated.
Now, only the current network is checked, but just in the case that `InputFee` component is active, because they are only used there.

This also improves the `Synchronizer` service, so, from now on, we could disable completely some actions until needed.

## Types of changes
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [x] Test (adding missing tests or fixing existing tests)

## Checklist
- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

luciorubeens commented on src/renderer/components/Input/InputFee.vue:
```diff
@@ -203,50 +224,28 @@ export default {
-      avgFee = avgFee * 1e-8
-      this.$set(this.feeChoices, 'AVERAGE', avgFee < this.maxV1fee ? avgFee : this.maxV1fee)
+    // Fees should be synchronized only when this component is active
+    this.$synchronizer.focus('fees')
```
I think the `call` method is enough, these values are not updated with this frequency:

```suggestion
    this.$synchronizer.call('fees')
```

j-a-m-l commented:
@luciorubeens I've updated the minimum fee part. It was intended to support getting the minimum fee from the statistics, but, since we aren't doing it, it wasn't necessary at all.

About the other change, the idea was keeping the fees as updated as possible only when the `InputFee` is visible (when the component is destroyed, the synchronizer action is paused again). Possibly the majority of users wouldn't use it, but I've added that to not leave any loose ends.
